hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestMobStoreCompaction.java

```diff
@@ -158,6 +161,9 @@ public void testSmallerValue() throws Exception {
 
   /**
    * During compaction, the mob threshold size is changed.
+   * The test is no longer valid. Major MOB compaction must be triggered by User
+   * HRegion does not provide public API to trigger major-compaction by User
+   * This test will move to mob sub-package.  
```

Comment:
Why isn't the test just moved now?

Reply:
Test actually passes. Moved to mob sub-package